packages/doenetml/src/Viewer/renderers/graph.jsx

```diff
@@ -116,6 +116,24 @@ export default React.memo(function Graph(props) {
                 }
             }
         });
+
+        // If an object is below the navigation bar and it is clicked, both
+        // the navigation button and the object drag/action are triggered.
+        // We hack our way around this behaviour by preventDefault-ing on all the button events
+        // after the nav bar is created.
+        const preventDefault = (e) => {
+            e.preventDefault();
+        };
+        newBoard.containerObj
```

Comment:
At least on Ubuntu Chrome, these event listeners are not being added. A console.log of `newBoard.containerObj.querySelectorAll(".JXG_navigation_button")` is an empty list. Even a console.log of `document.querySelectorAll(".JXG_navigation_button")` is giving an empty list at the time this `useEffect()` is called,

Reply:
It should work if you move it after the call to `addNavigationButtons` in the `[board]` `useEffect` around line 221.

Reply:
Putting the callbacks in the `[board]` `useEffect` does make the callbacks get called when clicking the navigation buttons. But, it doesn't prevent the clicks from propagating to the graph.

Here's the DoenetML I'm using to test this:
```
<graph>
    <point name="P" x="3" y="1" />
    <rectangle width="40" height="40" center="(0,0)" filled name="R" />
</graph>
<number name="n">1</number>
<updateValue triggerWhenObjectsClicked="P R" newValue="$n+1" target="n" />
```
The idea is that the counter `n` should not increment when a navigation button is clicked even if the rectangle is under the navigation buttons.

Reply:
The reason this isn't working is that the `mouseup` event on the button is not being triggered, and `triggerWhenObjectsClicked` is called with the up handler on the rectangle.

Reply:
Do you have a proposed fix?

Reply:
I could complete replace the nav with one that we control so that all events can be properly captured. Would it need anything other than a home and +/- buttons?

Reply:
We could follow the lead of jsxgraph and geogebra and just have the home and +/- buttons. In the example I gave, where a rectangle covered the whole graph, one cannot pan without the error buttons, as attempting to pan just moves the rectangle. But, I'm not sure how often that would happen.